manifests/4.4/image-references

```diff
@@ -15,3 +15,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-oauth-proxy:latest
+  - name: kibana
+    from:
+      kind: DockerImage
+      name: "quay.io/openshift/origin-logging-kibana5:latest"
```

Comment:
@jcantrill what is the ordering for this pr merging again? will it be after the feature-es6x branch?

Reply:
@igor-karpukhin this should be `kibana6`